src/shorthands.jl

```diff
@@ -5,51 +5,48 @@ function ylims! end
 function zlims! end
 
 """
-    xlabel!([scene,] xlabel)
+    xlabel!([axis,] xlabel)
 
-Set the x-axis label for the given Scene.
-Defaults to using the current Scene.
+Set the x-axis label for the given Axis.
+Defaults to using the current Axis.
 """
-function xlabel!(scene, xlabel::AbstractString)
-    axis = scene[OldAxis]
-    @assert !isnothing(axis) "The Scene does not have an axis!"
-    scene[OldAxis][:names][:axisnames][] = (xlabel, scene[OldAxis][:names][:axisnames][][2:end]...)
-    nothing
+function xlabel!(axis::Axis, xlabel::AbstractString)
+    axis.xlabel = xlabel
 end
 
+xlabel(figure::Figure, xlabel::AbstractString) = xlabel!(current_axis(figure), xlabel)
+xlabel(xlabel::AbstractString) = xlabel!(current_axis(), xlabel)
```

Comment:
IMO these functions should be `xlabel!` also (instead of having `xlabel` and `xlabel!` both).  Same for the y and z equivalents.